### PR TITLE
5495 Chapter 2 -- Benefit selection page

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/684

image
(There will, of course, be more than 3 chapters. 😛 )

image

Also, the Learn More links were backwards on the 5490 (where I copied them from), so I fixed those and shared the labels.

### CODE REVIEW

File: src/js/edu-benefits/5495/config/form.js [hunk 1]
```diff
@@ -48,6 +53,31 @@ const formConfig = {
...
+    },
+    benefitSelection: {
+      title: 'Benefit Selection',
+      pages: {
+        benefitSelection: {
+          path: 'benefits/selection', // other forms this is benefits/eligibility
+          title: 'Benefit selection',
+          uiSchema: {
+            benefit: {
+              'ui:title': 'Select the benefit that is the best match for you.',
```
Comment (jbalboni) [inline src/js/edu-benefits/5495/config/form.js]: There is a colon instead of a period at the end on the 5490.

File: test/edu-benefits/5495/config/benefitSelection.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  it('should show no errors when are empty', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          onSubmit={onSubmit}
+          data={{}}
+          state={{
+            benefitSelection: {
```
Comment (jbalboni) [inline test/edu-benefits/5495/config/benefitSelection.unit.spec.jsx]: Why is this necessary?

File: src/js/edu-benefits/5495/config/form.js [hunk 3]
```diff
@@ -48,6 +53,31 @@ const formConfig = {
...
+              'ui:widget': 'radio',
+              'ui:options': {
+                labels: survivorBenefitsLabels
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: [],
```
Comment (jbalboni) [inline src/js/edu-benefits/5495/config/form.js]: Can probably just delete this for now.

**Review by jbalboni — APPROVED** [review]
LGTM

File: test/edu-benefits/5495/config/benefitSelection.unit.spec.jsx (hunk 2 above)
Comment (cvalarida) [inline test/edu-benefits/5495/config/benefitSelection.unit.spec.jsx]: Totally isn't. Just forgot to remove it after copying.

**Review by annekainicUSDS — APPROVED** [review]
